docs/ToolsAndExamples/SDKExamples/OnchainMetadata/objectives.md

```diff
@@ -0,0 +1,25 @@
+---
+sidebar_position: 1
+---
```

Comment:
Can we get an estimate of how long the guide will take someone and add to the top here?

Reply:
From the perspective of someone with no knowledge of Lit, I think this guide will take at least 1 hour to read through. 

Let's add in: 
Estimated time to complete this guide: 1 hour